install.sh

```diff
@@ -114,9 +114,6 @@ source start_goto || check_status
 # add your first project
 project add goto || check_status
 
-# set the context to this project
```

Comment:
Have you tried reinstalling goto after these changes? 

It is important that the install script always work on linux / osx / windows (git-bash).

But, the goto project is intentionally added for demonstrative purposes, and good humor. "Never goto goto" is a catch phrase commonly used for instance ^^.

I don't  think we should remove the goto project from the setup. 
But I am more then happy to discuss how we could communicate more clearly that it is a demo project (but also would let you hack goto straight away). 

The pattern used by goto is to provide quite verbose texts that are giving clear advice to newcomers. (Many will actually install goto without much terminal experience).  

deactivate is a welcome feature - goto will remain as the initial project I think- but it will not be treated as a default project.